docs/apache-airflow/core-concepts/dags.rst

```diff
@@ -924,3 +924,10 @@ if it fails for ``N`` number of times consecutively.
 we can also provide and override these configuration from DAG argument:
 
 - ``max_consecutive_failed_dag_runs``: Overrides :ref:`config:core__max_consecutive_failed_dag_runs_per_dag`.
+
+Disable deletion of stale dags
+------------------------------
+
+In a versioned DAG context, there may be a need to run two versions of DAGs in parallel in two versions of workers.
+It can also be useful to keep the allowed DAGs if they are still in progress in the worker (n -1).
+To keep the allowed DAGs, you can change the value of the variable ``AIRFLOW__CORE__DISABLE_SCAN_STALE_DAGS`` to ``True``. By default, it is set to ``False``.
```

Comment:
Mention that we can also set in config.yml

Reply:
fixed